packages/ra-core/src/dataProvider/replyWithCache.ts

```diff
@@ -0,0 +1,61 @@
+import {
+    GetListParams,
+    GetListResult,
+    GetOneParams,
+    GetOneResult,
+    GetManyParams,
+    GetManyResult,
+} from '../types';
+
+export const canReplyWithCache = (type, payload, resourceState) => {
+    const now = new Date();
+    switch (type) {
+        case 'getList':
+            return (
+                resourceState &&
```

Comment:
could use lodash/get here:
```js
get(resourceState, ['list', 'validity', JSON.stringify(payload as GetListParams)]);
```

Reply:
Due to prettier, the code isn't super shorter, and due to lodash, it isn't more readable:

```jsx
            // with lodash
            return (
                get(resourceState, [
                    'list',
                    'validity',
                    JSON.stringify(payload as GetListParams),
                ]) > now
            );

            // without lodash
            return (
                resourceState &&
                resourceState.list &&
                resourceState.list.validity &&
                resourceState.list.validity[
                    JSON.stringify(payload as GetListParams)
                ] > now
            );
```

In this case, I prefer a bit more verbosity with a bit less indirection.